cmd/rqlite/main.go

```diff
@@ -127,7 +129,20 @@ func sendRequest(ctx *cli.Context, urlStr string, line string, argv *argT, ret i
 
 	nRedirect := 0
 	for {
-		resp, err := client.Post(url, "application/json", strings.NewReader(data))
+		req, err := http.NewRequest("POST", url, strings.NewReader(data))
+		if err != nil {
+			return err
+		}
+
+		if argv.Credentials != "" {
```

Comment:
`strings.TrimSpace` ?

Reply:
No need for this. Spaces have already been trimmed at this stage from my tests.

Reply:
Oh, great then